content/docs/notebooks/setup.md

```diff
@@ -0,0 +1,251 @@
+
+    <img src="/docs/images/new-notebook-server.png" 
+      alt="Form for adding a Kubeflow notebook server"
+      class="mt-3 mb-3 border border-info rounded">
+
+1. Enter a **name** of your choice for the notebook server. The name can
+  include letters and numbers, but no spaces. For example, `my-first-notebook`.
+1. Enter a **namespace** to identify the project group or team to which this 
+  notebook server belongs. The default is `kubeflow`.
+
+1. Select a Docker **image** for the baseline deployment of your notebook 
+  server. You can choose from a range of *standard* images or specify a 
+  *custom* image:
+
+  * **Standard**: The standard Docker images include typical machine learning 
+    (ML) packages that you can use within your Jupyter notebooks on 
+    this notebook server. Select an image from the **Image** dropdown menu.
+    The image names indicate the following choices:
+
+      * A TensorFlow version (for example, `tensorflow-1.13.1`). Kubeflow offers
+        a CPU and a GPU image for each minor version of TensorFlow.
+      * `cpu` or `gpu`, depending on whether you want to train your model on a CPU 
+        or a GPU. 
+        
+          * If you choose a GPU image, make sure that you have GPUs 
+            available in your Kubeflow cluster. Run the following command to check 
+            if there are any GPUs available:
+            `kubectl get nodes "-o=custom-columns=NAME:.metadata.name,GPU:.status.allocatable.nvidia\.com/gpu"`
+          * If you have GPUs available, you can schedule your server on a GPU node 
+            in the **Extra Resources** section at the bottom of the form. For 
+            example, to reserve two GPUs, enter the following JSON code: 
+          `{"nvidia.com/gpu": 2}`
+      * Kubeflow version (for example, `v0.5.0`).
+    
+
+  * **Custom**: If you select the custom option, you must specify a Docker image 
+    in  the form `registry/image:tag`. For guidelines on creating a Docker
+    image for your notebook, see the guide to 
+    [creating a custom Jupyter image](/docs/notebooks/custom-notebook/).
+
+1. Specify the total amount of **CPU** that your notebook server should reserve. 
+  The default is `0.5`. For CPU-intensive jobs, you can choose more than one CPU 
+  (for example, `1.5`).
+
+1. Specify the total amount of **memory** (RAM) that your notebook server should 
+  reserve. The default is `1.0Gi`.
+
+1. Specify a **workspace volume** to hold your personal workspace for this
+  notebook server. Kubeflow provisions a 
+  [Kubernetes persistent volume (PV)](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) for your workspace volume. The PV ensures that you can
+  retain data even if you destroy your notebook server.
+
+  * The default is to create a new volume for your workspace with the
+    following configuration:
+  
+      * Name: `notebook-workspace`
+      * Size: `10Gi`
+      * Mount path: `/home/jovyan`
+      * Access mode: `ReadWriteOnce`
```

Comment:
Do you think that it would be helpful to explain the access mode options? Or do you think that users will understand that this is explained in the Kubernetes PV doc?

Reply:
Thanks. I've explained the specific default setting, and linked again to the Kubernetes docs for more details.